File: src/main/java/com/adamkoch/iptables/matches/StartDateTimeExtensionMatchOption.java
package com.adamkoch.iptables.matches;

import java.time.temporal.Temporal;

public class StartDateTimeExtensionMatchOption extends DateTimeExtensionMatchOption {

  public StartDateTimeExtensionMatchOption(final Temporal startTemporal) {
    super(startTemporal);
  }

  @Override
  public String asString() {
    return super.asString(true);
  }
}

File: src/main/java/com/adamkoch/iptables/matches/StartDateTimeExtensionMatchOption.kt
package com.adamkoch.iptables.matches

import java.time.temporal.Temporal
import com.adamkoch.iptables.matches.DateTimeExtensionMatchOption

class StartDateTimeExtensionMatchOption(startTemporal: Temporal) :
    DateTimeExtensionMatchOption(startTemporal) {
    override fun asString(): String {
        return super.asString(true)
    }
}